test/scales/timeScaleTests.ts

```diff
@@ -15,24 +15,14 @@ describe("TimeScale tests", () => {
       var time2 = scale.domain()[1].valueOf();
       assert.strictEqual(time2, secondDate, "first value of domain set correctly");
     }
-    checkDomain(["10/1/2014", "11/1/2014"]);
-    checkDomain(["October 1, 2014", "November 1, 2014"]);
-    checkDomain(["Oct 1, 2014", "Nov 1, 2014"]);
+    checkDomain([new Date("10/1/2014"), new Date("11/1/2014")]);
+    checkDomain([new Date("October 1, 2014"), new Date("November 1, 2014")]);
+    checkDomain([new Date("Oct 1, 2014"), new Date("Nov 1, 2014")]);
```

Comment:
Unsure if we still want this test. It is actually checking JS coercing behavior (which we should assume correct)